parcels/examples/tutorial_parcels_structure.ipynb

```diff
@@ -0,0 +1,369 @@
+    "3. The timestep `dt` at which to execute the kernels\n",
+    "4. (Optional) The `ParticleFile` object to write the output to."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO: Compiled JITParticleAdvectionRK4 ==> C:\\Users\\GEBRUI~1\\AppData\\Local\\Temp\\parcels-tmp\\d8d64a9f5d2109ee91c0cae898d78aec_0.dll\n"
+     ]
+    }
+   ],
+   "source": [
+    "output_file = pset.ParticleFile(name=\"GCParticles.nc\", outputdt=3600) # the file name and the time step of the outputs\n",
+    "\n",
+    "pset.execute(AdvectionRK4,                 # the kernel (which defines how particles move)\n",
+    "             runtime=86400*6,              # the total length of the run in seconds\n",
+    "             dt=300,                       # the timestep of the kernel\n",
+    "             output_file=output_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One of the most powerful features of Parcels is the ability to write custom Kernels (see e.g. [this example](http://nbviewer.jupyter.org/github/OceanParcels/parcels/blob/master/parcels/examples/parcels_tutorial.ipynb#Adding-a-custom-behaviour-kernel) in the parcels tutorial). You probably want to define these kernels in this section; after defining your [`ParticleSet`](#ParticleSet) and before executing them. If your kernels become very large or complex, you might want to store them in another file and import them into your simulation script."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO: Compiled JITParticleAdvectionRK4WestVel ==> C:\\Users\\GEBRUI~1\\AppData\\Local\\Temp\\parcels-tmp\\cc034f200ae6700c99236a9e989972f8_0.dll\n"
+     ]
+    }
+   ],
+   "source": [
+    "def WestVel(particle, fieldset, time):\n",
+    "    if time > 86400 and time < 2*86400:\n",
+    "        uvel = -0.02\n",
+    "        particle.lon += uvel * particle.dt\n",
+    "        \n",
+    "WV_kernel = pset.Kernel(WestVel)\n",
+    "        \n",
+    "output_file = pset.ParticleFile(name=\"GC_WestVel.nc\", outputdt=3600)\n",
+    "\n",
+    "pset.execute(AdvectionRK4 + WV_kernel,     # simply add kernels using the + operator\n",
+    "             runtime=86400*6,              # the total length of the run in seconds\n",
+    "             dt=300,                       # the timestep of the kernel\n",
+    "             output_file=output_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "However, there are some key limitations to the Kernels that everyone who wants to write their own should be aware of:\n",
```

Comment:
Is it possible to add an 'anchor' to the notebook here, so that we can quickly link to this specific cell if someone has a question? I know you can do that if you provide a title to the cell, but not sure if it's also possible without a title

Reply:
I think I could either make a small title or add an 'empty' markdown cell above this with an html anchor tag:
https://stackoverflow.com/questions/28080066/how-to-reference-a-ipython-notebook-cell-in-markdown